StartioAdmobMediation/StartioAdmobAdAdapter.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022 Start.io Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import Foundation;
+
+@class GADMediationAdConfiguration;
+@class StartioAdmobParameters;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol StartioAdmobAdAdapter <NSObject>
+- (void)loadAdForAdConfiguration:(nonnull GADMediationAdConfiguration *)adConfiguration
```

Comment:
no need to add nonnull declaration as code is inside NS_ASSUME_NONNULL_BEGIN/NS_ASSUME_NONNULL_END macros